airflow/ui/src/pages/DagsList/DagsList.test.tsx

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import "@testing-library/jest-dom";
+import { render, screen, waitFor } from "@testing-library/react";
+import { setupServer, type SetupServerApi } from "msw/node";
+import { afterEach, describe, it, expect, beforeAll, afterAll } from "vitest";
+
+import { handlers } from "src/mocks/handlers";
+import { AppWrapper } from "src/utils/AppWrapper";
+
+let server: SetupServerApi;
+
+beforeAll(() => {
+  server = setupServer(...handlers);
+  server.listen({ onUnhandledRequest: "bypass" });
+});
+
+afterEach(() => server.resetHandlers());
+afterAll(() => server.close());
+
+describe("Dag Filters", () => {
+  it("Filter by selected last run state", async () => {
+    render(<AppWrapper initialEntries={["/dags"]} />);
+
+    await waitFor(() => expect(screen.getByTestId("dags-success-filter")).toBeInTheDocument());
+    await waitFor(() => screen.getByTestId("dags-success-filter").click());
+    await waitFor(() => expect(screen.getByText(/tutorial_taskflow_api_success/iu)).toBeInTheDocument());
```

Comment:
Where is the suffix `/iu` coming from ?

I was expecting just the `/tutorial_taskflow_api_success` to be in the document.

Reply:
I used regex during testing but made it string as i (case-insensitive), u (unicode) are not needed.

Reply:
Got it, thanks.